public/javascript/index.js

```diff
@@ -1,23 +1,97 @@
+		const id = document.getElementById('character-id').value;
+		charactersAPI
+			.getOneRegister(id)
+			.then((response) => {
+				document.querySelector('.characters-container').innerHTML = '';
+				createCharacterBox(response.data);
 
-  document.getElementById('fetch-one').addEventListener('click', function (event) {
+				document.querySelector('.edit-id').value = response.data.id;
+				document.querySelector('.edit-name').value = response.data.name;
+				document.querySelector('.edit-occupation').value = response.data.occupation;
+				document.querySelector('.edit-weapon').value = response.data.weapon;
+				document.querySelector('.edit-cartoon').checked = response.data.cartoon;
+			})
+			.catch((error) => console.log(error));
+	});
 
-  });
+	document.getElementById('delete-one').addEventListener('click', function(event) {
+		event.preventDefault();
+		const id = document.getElementById('character-id-delete').value;
+		charactersAPI
+			.deleteOneRegister(id)
+			.then((response) => {
+				document.getElementById('delete-one').classList.add('active');
+			})
+			.catch((error) => {
+				console.log(error);
+				document.getElementById('delete-one').classList.add('error');
+			});
+	});
 
-  document.getElementById('delete-one').addEventListener('click', function (event) {
+	document.getElementById('edit-character-form').addEventListener('submit', function(event) {
+		event.preventDefault();
 
-  });
+		const updateCharacter = {};
+		id = document.querySelector('.edit-id').value;
+		updateCharacter.name = document.querySelector('.edit-name').value;
+		updateCharacter.occupation = document.querySelector('.edit-occupation').value;
+		updateCharacter.weapon = document.querySelector('.edit-weapon').value;
+		updateCharacter.cartoon = document.querySelector('.edit-cartoon').checked;
 
-  document.getElementById('edit-character-form').addEventListener('submit', function (event) {
+		charactersAPI
+			.updateOneRegister(id, updateCharacter)
+			.then((response) => {
+				document.getElementById('edit-send-data').classList.add('active');
+			})
+			.catch((error) => {
+				console.log(error);
+				document.getElementById('edit-send-data').classList.add('error');
+			});
+	});
 
-  });
+	document.getElementById('new-character-form').addEventListener('submit', function(event) {
+		event.preventDefault();
 
-  document.getElementById('new-character-form').addEventListener('submit', function (event) {
+		const newCharacter = {};
+		newCharacter.name = document.querySelector('.create-name').value;
+		newCharacter.occupation = document.querySelector('.create-occupation').value;
+		newCharacter.weapon = document.querySelector('.create-weapon').value;
+		newCharacter.cartoon = document.querySelector('.create-cartoon').checked;
 
-  });
+		charactersAPI
+			.createOneRegister(newCharacter)
+			.then((response) => {
+				document.getElementById('create-send-data').classList.add('active');
```

Comment:
Só faltou ajustar o ID do botão de criação para a mudança do background funcionar nele também

```suggestion
				document.getElementById('send-data').classList.add('active');
```